x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -14878,7 +14878,7 @@
     "xpack.ingestPipelines.processors.defaultDescription.dateIndexName.indexNamePrefixDefault.noPrefixValueLabel": "プレフィックスなし",
     "xpack.ingestPipelines.processors.defaultDescription.dateIndexName.indexNamePrefixDefault.prefixValueLabel": "プレフィックス\"{prefix}\"を使用",
     "xpack.ingestPipelines.processors.defaultDescription.dissect": "分離したパターンと一致する値を\"{field}\"から抽出します",
-    "xpack.ingestPipelines.processors.defaultDescription.dot_expander": "\"{field}\"をオブジェクトフィールドに拡張します",
+    "xpack.ingestPipelines.processors.defaultDescription.dot_expander.dot_notation": "\"{field}\"をオブジェクトフィールドに拡張します",
```

Comment:
We don't typically update the translation files manually. Instead, please run the following and commit the result: 

```
node scripts/i18n_check --fix 
```

See: [elastic.co/guide/en/kibana/current/external-plugin-localization.html#_checking_i18n_messages](https://www.elastic.co/guide/en/kibana/current/external-plugin-localization.html#_checking_i18n_messages)

Reply:
I have run the above command in root. All the checks passed, and there is no change in any file, so nothing to commit. :slightly_smiling_face: 
I hope I did it right this time :sweat_smile: 
Thanks again!